packages/next/src/build/index.ts

```diff
@@ -1105,7 +1135,7 @@ export default async function build(
       )
 
       const routesManifestPath = path.join(distDir, ROUTES_MANIFEST)
-      const routesManifest: RoutesManifest = nextBuildSpan
+      let routesManifest: RoutesManifest = nextBuildSpan
```

Comment:
might be missing something in the diff but seems like this line and 1819 can keep the manifests as `const`

Reply:
Oh yup leftover from previous strategy